### DISCUSSION TITLE
Ignore Folders / Paths From Blitz.js Build

### DISCUSSION THREAD

**Discussion by burnfyi** [blitz-js/blitz#2459] [Q&A]
Given a folder structure like so:

```
- app
- db
- public
- ...
- server
```

what would be the best way to tell blitz.js to ignore the `server` folder when building? Is there an option to do so in the `blitz.config.js` file?

**Comment by flybayer** [comment]
Can you clarify what you mean by "ignore"? Like do you want them to not be processed by webpack or something?

**Reply by burnfyi** [reply]
Yep, like not processed by webpack

**Reply by flybayer** [reply]
Ok then yes you can edit the webpack config like this.

```ts
module.exports = {
    webpack: (config, { isServer }) => {
      if (!isServer) {
        config.module = config.module || {}
        config.module.rules = config.module.rules || []
        // Paths to ignore by webpack
        const excluded = [
          /utils\/logger/,
          /config\/serverEnv/,
          /integrations\/stripe\/server/,
        ]
        excluded.forEach((excluded) => {
          config.module.rules.push({ test: excluded, use: { loader: "null-loader" } })
        })
      }

      return config
    },

}
```

Note that the above config only ignores certain paths when `!isServer`. So you can adjust as needed to ignore on client or server or both.

**Reply by burnfyi** [reply]
awesome thanks!
